_posts/2016-09-14-resources.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Resources"
+date:   2016-09-14
+categories:  resources
+---
+
+## Websites
+Web Accessibility in Mind <a href="http://webaim.org/">Webaim</a>
+
+## Courses
+- Web Accessibility: Developing with Empathy <a href="https://www.udacity.com/course/web-accessibility--ud891">Udacity</a>
```

Comment:
I use Markdown pretty religiously but I still cant remember how to add links or images. Thanks!